src/tasks/upload-paportal/upload-paportal-v2/index.ts

```diff
@@ -29,5 +29,6 @@ export async function main(): Promise<void> {
     environmentUrl: getEnvironmentUrl(),
     path: parameterMap['UploadPath'],
     deploymentProfile: parameterMap['DeploymentProfile'],
+    modelVersion : parameterMap['ModelVersion'],
```

Comment:
You'll also need to add the parameter to the `tasks.json` in the same directory.

Reply:
Added